setup.cfg

```
@@ -17,7 +17,9 @@ long_description_content_type = text/x-rst
 maintainer = aiohttp team <team@aiohttp.org>
 maintainer_email = team@aiohttp.org
 license = Apache-2.0
```

```suggestion
license = Apache-2.0 AND MIT
```

The vendored llhttp is MIT, this should be reflected in the metadata too, if decided to include the license file.